src/RouteConfig.js

```diff
@@ -136,13 +136,15 @@ export default (
             if (functionResult.then) {
               const result = await Promise.race([
                 functionResult,
-                new Promise(
-                  (resolve: () => void, reject: () => void): number =>
-                    setTimeout(
-                      () => reject(new Error('timeout')),
-                      settings.API_TIMEOUT * 1000,
-                    ),
-                ),
+                !serverSentEvents
+                  ? new Promise(
```

Comment:
@jlkalberer that `timeout` middleware is doing bad with async and almost every issue in their repo about `Can't set headers after they are sent.`..and what they offer: manually reject our promises/timeouts in requests when `req.on('timeout')`...
so I prefer use your custom approach instead, and just pass condition here.